.github/workflows/nodejs.condo.ci.yml

```
@@ -936,3 +940,51 @@ jobs:
           PROMETHEUS_USER: ${{ secrets.K6_PROMETHEUS_USER }}
           PROMETHEUS_PASSWORD: ${{ secrets.K6_PROMETHEUS_PASSWORD }}
           GH_REF_NAME: ${{ github.event.pull_request.head.ref || github.ref_name }}
+
+  run-resident-app-tests:
+    name: Resident app tests
+    runs-on: ubuntu-22.04
+    needs:
+      - authorize
+      - build-image
+      - detect-changes
+    if: ${{ needs.detect-changes.outputs.residentApp == 'true' }}
```

actionlint (1.7.8)

property "residentapp" is not defined in object type {accruals-gateway: string; address-service: string; dev-portal-api: string; eps: string; external-api: string; meter-importer: string; migrator: string; miniapp-utils: string; open-condo: string; pos-integration: string; rb: string; registry-importer: string; resident-app: string; webhooks: string}

(expression)